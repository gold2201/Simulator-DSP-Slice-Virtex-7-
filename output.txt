P=000000000000000000000000000000000000000000000002 CARRYOUT=0 OVERFLOW=0
