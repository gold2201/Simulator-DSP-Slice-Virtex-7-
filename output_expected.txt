P=000000000000000000000000000001111111111111011111 CARRY=0 OVER=0 PAT=0 UNDER=0
